themes/bootstrap3/theme.config.php

```diff
@@ -61,5 +61,42 @@
             'layoutClass' => 'VuFind\View\Helper\Bootstrap3\LayoutClass',
             'search' => 'VuFind\View\Helper\Bootstrap3\Search'
         ]
+    ],
+    'icons' => [
+        'defaultSet' => 'FontAwesome',
+        'sets' => [
+            /**
+             * Define icon sets here.
+             *
+             * All sets need:
+             * - 'template': which template the icon renders with
+             * - 'src': the location of the relevant resource (font, css, images)
+             * - 'prefix': prefix to place before each icon name for convenience
+             *             (ie. fa fa- for FontAwesome, default "")
+             */
+            'FontAwesome' => [
+                'template' => 'font',
+                'prefix' => 'fa fa-',
+                'src' => 'vendor/font-awesome.min.css',
+            ],
+            'Remix' => [
+                'template' => 'svg-sprite',
+                'prefix' => 'ri-',
+                'src' => 'remixicon.symbol.svg',
+            ],
+        ],
+        'aliases' => [
+            /**
+             * Icons can be assigned or overriden here
+             *
+             * Format: 'icon' => [set:]icon
+             *
+             * Icons assigned without set will use the defaultSet (above)
+             */
+            'favorites' => 'Remix:star-fill',
+            'my-account' => 'user-circle',
+            'search' => 'Remix:search-line',
```

Comment:
I think I already mentioned this in a different thread, but just to highlight it: do we intend to use Remix here "for real" or is this just an example? If it's an example, we should remove it or comment it out. If it's for real, we should add Remix to our icons wiki page. I've added a TODO checkbox to track the need to decide one way or the other.

Reply:
I do intend to make a Remix icon set but in the other PR.

Reply:
Thanks!